pkgs/by-name/ma/matugen/package.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, stdenv
+, darwin
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "matugen";
+  version = "2.2.0";
+
+  src = fetchFromGitHub {
+    owner = "InioX";
+    repo = "matugen";
+    rev = "v${version}";
+    hash = "sha256-4VX7Rt+ftEH8nwg59eT7TsvHYUf8/euUmwh/JLc4rLc=";
+  };
+
+  cargoHash = "sha256-axBdOzCUwseV2ltgarmIS3IOYLX3q3rXeA2y6y7aNFI=";
+
+  buildInputs = lib.optionals stdenv.isDarwin [
+    darwin.apple_sdk.frameworks.Security
+  ];
```

```suggestion
    darwin.apple_sdk.frameworks.SystemConfiguration
  ];
```

```
       >   = note: ld: framework not found SystemConfiguration
       >           clang-16: error: linker command failed with exit code 1 (use -v to see invocation)
       >
       >
       > error: could not compile `matugen` (bin "matugen") due to 1 previous error
       For full logs, run 'nix-store -l /nix/store/2w6n9hvxgl0vm81j1q09j34v7irjybzz-matugen-2.2.0.drv'.
```

I tried to build matugen in aarch64-darwin with nixpkgs-unstable(master), but it threw the above error. It's better to add `darwin.apple_sdk.frameworks.System` to buildInputs.